R/PersonProfile.R

```diff
@@ -0,0 +1,115 @@
+# @file ROhdsiWebApi
+#
+# Copyright 2020 Observational Health Data Sciences and Informatics
+#
+# This file is part of ROhdsiWebApi
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#' Get person profile data
+#'
+#' @details
+#' Get a R object with person profile data. This function may be used for 
+#' visualizing a patients profile in tables or visualization.
+#'
+#' @param baseUrl        The base URL for the WebApi instance, for example:
+#'                       "http://server.org:80/WebAPI".
+#' @param sourceKey      The source key for a CDM instance in WebAPI, as defined in the
+#'                       Configuration page
+#' @param personId       The personId of the person whoose profile is being reviewed
+#' @param cohortId       <OPTIONAL> The cohortId of the cohort that the person belongs to
```

Comment:
What happens when I provide a cohortId? Do the cohort start and end date then become part of the patient profile? Please explain in the documentation.